src/vendor/ez_toml/v1.ml

```diff
@@ -38,7 +38,17 @@ module TOML = struct
           lexbuf.lex_curr_p <- {lexbuf.lex_curr_p with pos_fname = file}
     end;
     try
-      let lines = Internal_parser.toml Internal_lexer.tomlex lexbuf in
+      let eof = ref false in
```

Comment:
I though the problem was a missing newline expected by the parser. Is this actually fixing the issue?

Reply:
The idea is to always send a EOL token before the EOF token, so there is always an EOL at the end...

Reply:
And yes, I added a test in the original `ez_toml` and the problem is fixed there.

Reply:
Ah sure! I originally did not distinguish `EOF` and `EOL`.